### ISSUE TITLE
x-csrf-token won't set

### ISSUE THREAD

**Issue by AGR-DEVS** [nab0rs/Anatomy---Discord-Bot-Open-Sourced#1]
Hey nab0rs, I tried using your discord bot and after some time of debugging, I figured out that the line: session.headers['x-csrf-token'] = session.get('https://www.roblox.com/home').content.decode('utf8').split("Roblox.XsrfToken.setToken('")[1].split("');")[0]
doesn't actually work. Any help would be appreciated, thanks.

**Comment by NotSwaraj** [comment]
yeah roblox changed csrf-tokens around, do split "data-token" instead

**Comment by nab0rs** [comment]
> Hey nab0rs, I tried using your discord bot and after some time of debugging, I figured out that the line: session.headers['x-csrf-token'] = session.get('[https://www.roblox.com/home').content.decode('utf8').split("Roblox.XsrfToken.setToken('")[1].split("');")[0]](https://www.roblox.com/home').content.decode('utf8').split(%22Roblox.XsrfToken.setToken('%22)%5B1%5D.split(%22');%22)%5B0%5D)
> doesn't actually work. Any help would be appreciated, thanks.

been patched for a while now

**Comment by NotSwaraj** [comment]
its not patched, they just changed where csrf tokens are placed out on a
page

**Comment by nab0rs** [comment]
> page

they did patch. They removed it from the html and they decided to put a lot of ratelimit
